File: C/soma_vetor.c
#include <stdio.h>

int main() {
    
    double soma = 0, media = 0;
    int n, i;
    
    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &n);
    
    double vet[n];
    
    for (i = 0; i < n; i++) {
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
    }
    
    printf("VALORES =  ");
    for (i = 0; i < n; i++){
        soma = (soma + vet[i]);
        media = (soma / n);
        printf("%.1lf ", vet[i]);
    
    }
    
    printf("\nSOMA = %.1lf", soma);
    printf("\nMEDIA = %.1lf", media);
    
    return 0;
}

File: JAVA/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		try (Scanner sc = new Scanner(System.in)) {
			int N;
			int i;
			double soma = 0;
			double media = 0;
			
			System.out.println("Quantos numeros você vai digitar? ");
			N = sc.nextInt();
			
			double[] vet = new double[N];
			
			for (i = 0; i < N; i++) {
				System.out.print("Digite um número: ");
				vet[i] = sc.nextDouble();
			}
			
			System.out.print("VALORES = ");
			
			for (i = 0; i < N; i++) {
				soma += vet[i];
				media = (soma / N);
				System.out.printf("%.1f ", vet[i]);
			}
			
			System.out.printf("\nSOMA = %.2f%n", soma);
			
			System.out.printf("MEDIA = %.2f%n",  media);
			
			
			sc.close();
		}
	}

}
